Camera.MAUI/Camera.MAUI.csproj

```diff
@@ -53,6 +58,7 @@
 	</ItemGroup>
 
 	<ItemGroup>
+		<PackageReference Include="CommunityToolkit.Mvvm" Version="8.4.0" />
```

Comment:
I don't like adding a new dependency like this, but was quickest way to make it bindable.  (`ObservableObject : INotfyPropertyChanged`)  could make some impl internal to support `INotfyPropertyChanged` without the dependency.